modules/custom/az_card/src/Plugin/Field/FieldWidget/AZCardWidget.php

```diff
@@ -168,7 +169,7 @@ public function formElement(FieldItemListInterface $items, $delta, array $elemen
           '#type' => 'link',
           '#title' => $items[$delta]->link_title ?? '',
           '#url' => $link_url ? $link_url : '#',
-          '#attributes' => ['class' => ['btn', 'btn-default', 'w-100']],
```

Comment:
As part of the widget, there is a "preview container" that is constructed for the "collapsed" card. It looks like this needs to be updated to add the selected classes. I think this might be what is causing the issue @smayden describes.